src/JsonLd/Serializer/JsonLdEncoder.php

```diff
@@ -11,17 +11,32 @@
 
 namespace ApiPlatform\Core\JsonLd\Serializer;
 
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Serializer\Encoder\DecoderInterface;
+use Symfony\Component\Serializer\Encoder\EncoderInterface;
+use Symfony\Component\Serializer\Encoder\JsonDecode;
+use Symfony\Component\Serializer\Encoder\JsonEncode;
 use Symfony\Component\Serializer\Encoder\JsonEncoder;
 
 /**
  * JSON-LD Encoder.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class JsonLdEncoder extends JsonEncoder
+final class JsonLdEncoder implements EncoderInterface, DecoderInterface
```

Comment:
> Because we need to override supports* method

I don't quite understand that. Why can't we just set `$format` to `json`? Is there any benefit of using a different `jsonld` format?

> we tweak the options passed to JSON_ENCODE for RFC compliance

I thought that's what `$context['json_encode_options']` is for?